src/net/sf/memoranda/ui/EventsPanel.java

```diff
@@ -53,6 +56,11 @@
     JRadioButton dayRb = new JRadioButton("Today");
     JRadioButton weekRb = new JRadioButton("7 Days");
     JRadioButton monthRb = new JRadioButton("30 Days");
+    JRadioButton customRb = new JRadioButton("Custom Range");	// US-101
+    
+    // create text field and go button for custom user input US-101
+    JTextField customTextField = new JTextField(50);
+    JButton goButton = new JButton("Go");
```

Comment:
These should be constants (CG:6) (I noticed I've neglected this a lot in my code as well)